project/plugins.sbt

```diff
@@ -1,3 +1,4 @@
 addSbtPlugin("com.geirsson"  % "sbt-ci-release" % "1.5.5")
 addSbtPlugin("org.scoverage" % "sbt-scoverage"  % "1.6.1")
 addSbtPlugin("org.scoverage" % "sbt-coveralls"  % "1.2.7")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly"  % "0.15.0")
```

Comment:
Why do we need assembly? This is a library.

Reply:
Pretty sure `sbt assembly` was failing w/o this, but now we’re no longer using that command. I’ll remove it.